plugins/modules/aci_system_banner.py

```diff
@@ -269,6 +269,14 @@ def main():
     if state == "present":
         regex_url = "^https?:\\/\\/(?:www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b(?:[-a-zA-Z0-9()@:%_\\+.~#?&\\/=]*)$"
 
+        if gui_banner is not None:
+            if re.fullmatch(regex_url, gui_banner):
+                is_gui_message_text = "no"
+            else:
+                is_gui_message_text = "yes"
+        else:
+            is_gui_message_text = "yes"
+
```

Comment:
can be reduced by
if gui_banner is not None and re.fullmatch(regex_url, gui_banner):
     is_gui_message_text = "no"
else:
      is_gui_message_text = "yes"

Reply:
Yes it can! Thank you!